### PR TITLE
[CPU] Introduce LoRA macro operation

### PR DESCRIPTION
**Pull request by maxnick**
### Details:
To minimize the overheads on the LoRA subgraph operation processing (shape update, memory allocation, etc.) it does make sense to merge such subgraphs into a specific LoRA macro operation, which may exploit some LoRA properties to optimize performance. 

### Tickets:
 - CVS-153035
 - CVS-155112

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/lora.cpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    // for the output descriptors, use the configuration of the graph's output nodes
+    auto outputDescriptors = m_graph.getOutputMemoryDescriptors();
+
+    const auto& desc = outputDescriptors.front();
+
+    if (!desc->isCompatible(*(inConfs.front().getMemDesc()))) {
+        // just a sanity check
+        THROW_CPU_NODE_ERR("Unexpected input/output descriptor mismatch");
+    }
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/lora.cpp:71]: CPU_NODE_ASSERT

File: src/plugins/intel_cpu/src/nodes/lora.cpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    std::vector<PortConfig> outConfs;
+    outConfs.emplace_back(desc);
+    outConfs.front().inPlace(0); // use the memory from the first input inPlace
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/lora.cpp:75]: Better to specify as part of a construction

File: src/plugins/intel_cpu/src/nodes/lora.cpp (hunk 1 above)
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/lora.cpp:71]: CPU_NODE_ASSERT

File: src/plugins/intel_cpu/src/nodes/lora.cpp (hunk 2 above)
Comment (EgorDuplensky) [reply src/plugins/intel_cpu/src/nodes/lora.cpp:75]: Am I missing something, or this is basically the only difference in comparison to Composite node?

File: src/plugins/intel_cpu/src/nodes/lora.cpp (hunk 1 above)
Comment (maxnick) [reply src/plugins/intel_cpu/src/nodes/lora.cpp]: Done

File: src/plugins/intel_cpu/src/nodes/lora.cpp (hunk 2 above)
Comment (maxnick) [reply src/plugins/intel_cpu/src/nodes/lora.cpp]: > Better to specify as part of a construction

Done

File: src/plugins/intel_cpu/src/nodes/lora.cpp (hunk 1 above)
Comment (maxnick) [reply src/plugins/intel_cpu/src/nodes/lora.cpp]: Done

**Review by EgorDuplensky — APPROVED** [review]

**Comment by maxnick** [comment]
@EgorDuplensky, thank you!
@dmitry-gorokhov , the PR is ready for the final review.

**Pull request merged**